src/main/java/de/tum/in/www1/artemis/domain/GradingInstruction.java

```diff
@@ -0,0 +1,159 @@
+package de.tum.in.www1.artemis.domain;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import javax.persistence.*;
+
+import org.hibernate.annotations.Cache;
+import org.hibernate.annotations.CacheConcurrencyStrategy;
+
+/**
+ * A Structured Grading Instruction.
+ */
+@Entity
+@Table(name = "grading_instruction")
+@Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
+public class GradingInstruction implements Serializable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    // the score students get if this grading instruction is applicable
+    @Column(name = "credits")
+    private double credits;
+
+    // describes the level of performance (e.g. "poor" or "average" )
+    @Column(name = "grading_scale")
+    private String gradingScale;
```

Comment:
Could this be an enum?

Reply:
I discussed this with Stephan and we decided to leave it for now and maybe change it later to enum if it makes sense.